src/main/java/dev/task/dndquest/mapper/PlayerDtoMapper.java

```diff
@@ -14,6 +14,6 @@ public PlayerResponseDto matToDto(Player entity) {
 
     @Override
     public Player mapToEntity(PlayerRequestDto dto) {
-        return new Player(dto.getLogin(), dto.getPassword());
+        return new Player(null, dto.getLogin(), dto.getPassword());
```

Comment:
why null here?

Reply:
I deleted all unnecessary constructors and left only all fields constructor. In this case instead of null should be player id, that will be given by db during process of saving.

Reply:
Probably player entity will have more fields in the future, f.e. characters. But in this case i'll return from allArgsConstructor to the constructor only with login and pass.
So i wood like leave mapper.